model/service.go

```diff
@@ -201,6 +201,14 @@ type ServiceDiscovery interface {
 
 	// HostInstances lists service instances for a given set of IPv4 addresses.
 	HostInstances(addrs map[string]bool) []*ServiceInstance
+
+	// ManagementPorts lists set of management ports associated with an IPv4 address.
```

Comment:
Might be worth explaining what purpose these ports serve in istio context: the proxy should allow connections through this port for transparent proxy injection.